Marlin/Configuration.h

```diff
@@ -3383,6 +3383,10 @@
 // duty cycle is attained.
 //#define SOFT_PWM_DITHER
 
+//Use Additional Pwn fan headers for partcooling allowing up to 3 total part cooling fans that are controlled thru M106 & M107
+//#define EXTRA_PARTCOOLER_FAN1 //Also use Fan1 for part cooling 
+//#define EXTRA_PARTCOOLER_FAN2 //Also use Fan2 for part cooling
```

Comment:
Instead of hardcoding these to FAN1/FAN2, it'd be better if the user could specify which fan to use since modern boards have lots of fan headers.

Reply:
i could easily add any number of additional fan ports for use as part cooling. it would seem to me more then 4 would not be that useful. i'm open to ideas for a better implementation for sure, the way i have done it has been rather crude but effective when i was asked to come up with a way to do this. they where insistent on dual part cooling fans controlled in this way.